Description: do not include -lssl in LIBS
Origin: vendor
Forwarded: not-needed
Author: Damyan Ivanov <dmn@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2024-06-15

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -11,7 +11,7 @@
 if ($args =~ /LIBDIR[ =](\S+)/) {
   $libdir = $1;
 }
-my $libs = "-lssl -lcrypto";
+my $libs = "-lcrypto";
 
 if ( $Config{osname} eq 'aix' ) {
   $libs = $libs . ' -lz';
